lua/grpc.lua

```diff
@@ -44,6 +44,20 @@ GRPC.error = function(msg)
   }
 end
 
+--
+-- Logging methods
+--
+
+GRPC.logError = function(err)
+  grpc.log_error(err)
+  env.error("[GRPC] "..err)
+end
+
+GRPC.logWarning = function(err)
+  grpc.log_warning(err)
+  env.warning("[GRPC] "..err)
```

Comment:
Logs it into both `dcs.log` and `gRPC.log`. I'd also be fine though to remove the logs from `dcs.log`.

Reply:
I think having warnings and exceptions in the main dsc.log reduces work for server admins. Everything else being in grpc.log is fine I think.

Reply:
Oh, it just occurred to me. Can we keep the GRPC startup message appearing in DCS.log?

Reply:
Yes the startup messages (`loading ...` and `loaded ...`) are kept in the `dcs.log` - I also think that they are very important to even know if the gRPC module started or not, as we cannot even expect logs in the `gRPC.log` file otherwise.

Reply:
> I think having warnings and exceptions in the main dsc.log reduces work for server admins. Everything else being in grpc.log is fine I think.

Most errors and warnings would be mirrored into both files, but there are already errors that will only make it into the `gRPC.log` file. They are the ones that are logged on the Rust side. One of them being the fix for https://github.com/DCS-gRPC/rust-server/issues/19 as an example.

Reply:
My plan is btw to hook up the logging to https://sentry.io next, as I rarely skim logs for errors unless I've explicitly noticed something not working.

Reply:
Could we avoid hooking directly up to certain sinks?

If we can make sure that the server outputs JSON logs to file then another separate application can then ship those json logs to the sinks of the admins choice. I know that Hoggit uses an ELK stack for example.